cfme/tests/services/test_provision_stack.py

```diff
@@ -358,7 +377,8 @@ def test_error_message_azure():
                details.
         startsin: 5.8
     """
-    pass
+    provision_request = order_stack
+    assert not provision_request.is_succeeded()
```

Comment:
true... to double-check, I guess. The assertion in the fixture also inside the LogValidator.

So I removed `stack_not_created_error` because it's essentially the test steps, not the setup, so I moved it to the test itself and now there are no such repetitions. 

Thanks @john-dupuy !